src/TriggerBinding/SqlTableChangeMonitor.cs

```diff
@@ -124,21 +124,23 @@ internal sealed class SqlTableChangeMonitor<T> : IDisposable
             this._bracketedLeasesTableName = !string.IsNullOrEmpty(bracketedLeasesTableName) ? bracketedLeasesTableName : throw new ArgumentNullException(nameof(bracketedLeasesTableName));
             this._userTableColumns = userTableColumns ?? throw new ArgumentNullException(nameof(userTableColumns));
             this._primaryKeyColumns = primaryKeyColumns ?? throw new ArgumentNullException(nameof(primaryKeyColumns));
+            this._sqlOptions = sqlOptions ?? throw new ArgumentNullException(nameof(sqlOptions));
             this._executor = executor ?? throw new ArgumentNullException(nameof(executor));
             this._logger = logger ?? throw new ArgumentNullException(nameof(logger));
 
             this._userTableId = userTableId;
             this._telemetryProps = telemetryProps ?? new Dictionary<TelemetryPropertyName, string>();
 
             // Check if there's config settings to override the default max batch size/polling interval values
+            // TODO: remove reading from settings when we decide to move to reading them from func.json.
```

Comment:
I'd also suggest making it clear in our docs (both xml and docs) that if both are specified the app settings values will be chosen.